scenariolib/visit.go

```diff
@@ -134,11 +134,16 @@ func (v *Visit) ExecuteScenario(scenario Scenario, c *Config) error {
 		if v.WaitBetweenActions {
 			var timeToWait int
 			if c.TimeBetweenActions > 0 {
-				timeToWait = c.TimeBetweenActions
+				timeToWait = c.TimeBetweenActions //Random waiting time
+				WaitBetweenActions(timeToWait)
+			}else if c.TimeBetweenActions < 0 {
+				timeToWait = c.TimeBetweenActions*(-1)
+				time.Sleep(time.Duration(timeToWait) * time.Second) //Constant Waiting time
```

Comment:
Well you still don't specify that a positive TimeBetweenActions if random this way...

It was created to be random as that was the purpose of the bot. Can you just not set it to a small number and leave it being random? :) or do you need it to be constant for some reason?